lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module

```diff
@@ -696,3 +696,41 @@ function dosomething_helpers_get_campaign_status($start_date, $end_date) {
     return 'closed';
   }
 }
+
+/**
+ * Gets the current run for a given campaign nid and user.
+ *
+ * @param int $nid
+ * @param object $user
+ *  Optional. Will default to global $user if not supplied
```

Comment:
Optional — should be lowercase and in parentheses, see https://www.drupal.org/node/1354#param